Kronos-Shifts-Connector/Microsoft.Teams.Shifts.Integration/Microsoft.Teams.Shifts.Integration.API/Controllers/TeamsController.cs

```diff
@@ -149,11 +153,17 @@ public async Task<ActionResult> UpdateTeam([FromRoute] string aadGroupId)
                 { "IncomingAadGroupId", aadGroupId },
             };
 
+            // get the kronos org job mapping information
+            var teamMappings = await this.teamDepartmentMappingProvider.GetMappedTeamDetailsAsync(aadGroupId).ConfigureAwait(false);
+
+            // get the timezone for the first mapping - logically all mappings for the team must have the same time zone set
+            var timeZone = teamMappings[0].KronosTimeZone;
```

Comment:
what happens if teamMappings is null?

Reply:
Again have switched it to the same as the other controllers with the same caveat regarding mappings.